MBINCompiler/MBINCompiler.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0-windows;net7.0-windows</TargetFrameworks>
```

Comment:
you removed the `net7.0-windows` target here. You need to keep all the `net7.0` targets

Reply:
already added